protobuf/api/execution.proto

```diff
@@ -31,6 +31,9 @@ message StreamExecutionRequest{
     // Note: to filter multiple statues or flags, eg:
     // Failed | Completed
     definition.Status status = 1;
+
+    // hash is a hash for filtering execution by instance hash.
+    string hash = 2;
```

Comment:
can you rename to `instanceHash`.
We don't know if it executionHash, or whatever